src/properties/competitionbot/intake.properties

```diff
@@ -0,0 +1,4 @@
+dream.intake.sideBelt.forward = 1
+dream.intake.sideBelt.reverse = -1
+dream.intake.mainBelt.forward = 1
+dream.intake.mainBelt.reverse = -1
```

Comment:
Just a minor point...usually we end our files with a blank line. If anything, we just do it so GitHub gets rid of the little red arrow at the end of this line.